src/apps/reducers.js

```diff
@@ -45,8 +45,8 @@ function bundleInfo(state = defaultBundleInfo, action) {
         case FETCH_BUNDLE_ERROR:
             return {
                 bundleId: action.bundleId,
-                bundle: null,
-                inputs: null,
+                bundle: action.bundle,
+                inputs: state.inputs || [],
```

Comment:
I think its fine we're doing this, we just need to resolve the issues this will call with the run button (ie the run button used to get disabled when there was error because the bundle was set to null).